package.json

```diff
@@ -8,6 +8,9 @@
     "dist",
     "types"
   ],
+  "publishConfig": {
```

Comment:
Why did you add this?

Reply:
seems like this was an auto add by something in vscode. removed it in 9804e9d